id.html

```diff
@@ -0,0 +1,238 @@
+          </a>
+        </div>
+        <div>
+          <h1>Bkebi Genshin Cheat
+            <br>
+            <a href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases/latest"><img
+                src="https://img.shields.io/github/v/release/Bkebi-Group/Bkebi-GC-Release?style=for-the-badge"></a>
+            <a href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases"><img
+                src="https://img.shields.io/github/downloads/Bkebi-Group/Bkebi-GC-Release/total.svg?style=for-the-badge"></a>
+            <a href="https://discord.gg/bkebi"><img
+                src="https://img.shields.io/discord/1026295403282436097?label=Discord&logo=discord&style=for-the-badge&color=blueviolet"></a>
+          </h1>
+        </div>
+        <div>
+          <a id="download-button" class="btn"
+            href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases/latest">Dapatkan Sekarang
+            <i class="iconfont icon-xiazai"></i>
+          </a>
+          <a id="documentation-link" class="btn" href="docs/index.html" target="_blank">Dokumentasi 
+            <i class="iconfont icon-wendang"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+    <a id="down-arrow" href="#kali-highlights">
+      <div>
+      </div>
+    </a>
+  </header>
+  <section id="kali-highlights">
+    <div>
+      <div>
+        <h1>Gratis dan Canggih</h1>
+      </div>
+    </div>
+    <h3>Menghabiskan lebih sedikit energi dan waktu, membantu menyelesaikan pekerjaan mu, memperbaiki perngalaman game anda, dan tidak lagi menderita dengan akun yang burik.</h3>
+    <ul>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-youxiyouxisheji"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Player
+          </h4>
+        </a>
+        <p>Mode serangan ganda, untuk menghindari terdeteksi oleh server dari damage yang tinggi. Hapus cooldown, ledakan elemen tidak perlu menunggu cooldown, karena variasi permainan, server tidak akan memeriksanya. Mode tak terkalahkan probabilitas yang sudah di sesuaikan.</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-dituleiditu"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Dunia
+          </h4>
+        </a>
+        <p>Secara otomatis menyelesaikan tantangan dunia, memasak makanan secara otomatis, menghancurkan objek secara otomatis, memancing ikan secara otomatis, mengambil item secara otomatis, mengambil item dengan jangkuan yang jauh, fairy spirits secara otomatis, dialog NPC otomatis (dapat di percepat), menebang pohon secara otomatis, menyamarkan Waktu In-game, Cloing global, Kill Aura, Membuat makhluk adsorpsi, drop adsorpsi, dengan teleportasi otomatis agar tidak membuat tangan anda lelah, disesuaikan untuk hati anda.</p>
```

Comment:
change it to `Dapat menyelesaikan Tantangan secara otomatis di dunia, memasak makanan secara otomatis, menghancurkan suatu objek secara otomatis, mengambil item drop secara otomatis, mengambil item dari jauh, mengantarkan Seelie secara otomatis, dialog NPC secara otomatis (dapat dipercepat), menebang pohon secara otomatis, memalsukan waktu di game, global multiplier, Kill Aura, menarik entitas ke pemain, mengumpulkan drop, teleportasi otomatis agar tidak melelahkan, dan dapat disesuaikan sesuka hati.`